app/src/main/java/com/example/pocketwatching/Activities/ProfileActivity.java

```diff
@@ -311,6 +322,64 @@ public void onFailure(Call<BlockBalance> call, Throwable t) {
         });
     }
 
+    private void setupChart(List<Float> xValues, List<Float> yValues) {
+        XAxis xAxis = volumeReportChart.getXAxis();
+
+        volumeReportChart.getAxisLeft().setEnabled(false);
+        volumeReportChart.getAxisRight().setEnabled(false);
+        volumeReportChart.getAxisRight().setAxisMaximum(10);
+        volumeReportChart.getDescription().setEnabled(false);
+        volumeReportChart.setTouchEnabled(true);
+        volumeReportChart.setDragEnabled(true);
+        volumeReportChart.animateY(500, Easing.EaseInCubic);
+
+        XAxis.XAxisPosition position = XAxis.XAxisPosition.BOTTOM;
+        xAxis.setPosition(position);
+
+        xAxis.setValueFormatter(new ClaimsXAxisValueFormatter(xValues));
+
+        LineDataSet set1;
```

Comment:
Rename variable to something more descriptive and more easily understandable like ```portfolioValueDataSet```